### PR TITLE
Fix compile with disabled FEATURE_READYTORUN option

### PR DESCRIPTION
**Pull request by BredPet**
Signed-off-by: Petr Bred <bredpetr@gmail.com>

### CODE REVIEW

**Comment by BredPet** [comment]
@sergign60 
@Dmitri-Botcharnikov 
@jkotas please review

File: Documentation/building/linux-instructions.md [hunk 1]
```diff
@@ -237,3 +237,72 @@ If you want to focus on the speed optimization for high-end devices, you have to
  SET (CLR_DEFINES_DEBUG_INIT              DEBUG _DEBUG _DBG URTBLDENV_FRIENDLY=Checked BUILDENV_
 ```
 
+Additional configuring features for ARM/Linux
+=============================================
+
+Disable READYTORUN
+------------------
```
Comment (jkotas) [inline Documentation/building/linux-instructions.md]: I do not think this should be documented here like this. It will get out of date very quickly. If you have a good reason for compiling with this disabled often, make it option for the CMake scripts.

File: src/vm/eventpipe.cpp [hunk 2]
```diff
@@ -107,7 +107,7 @@ void EventPipe::Shutdown()
 
 void EventPipe::Enable(
     LPCWSTR strOutputPath,
-    uint circularBufferSizeInMB,
+    unsigned int circularBufferSizeInMB,
```
Comment (jkotas) [inline src/vm/eventpipe.cpp]: This looks unrelated to the change. Could you please submit separate PR for it?

File: Documentation/building/linux-instructions.md (hunk 1 above)
Comment (BredPet) [inline Documentation/building/linux-instructions.md]: Ok

File: src/vm/eventpipe.cpp (hunk 2 above)
Comment (BredPet) [inline src/vm/eventpipe.cpp]: The chain is 
`eventpipe.cpp -> common.h -> ceeload.h -> (IF FEATURE_READYTORUN -> readytoruninfo.h -> nativeformatreader.h -> uint)`
Of course, I can still separate, but it seems to me that this is a direct binding.

Comment (jkotas) [inline src/vm/eventpipe.cpp]: I see. It was not obvious to me - it is fine to keep as part of this.

**Comment by BredPet** [comment]
@jkotas please review
